sqlalchemy-stubs/sql/schema.pyi

```diff
@@ -397,53 +520,56 @@ class MetaData(SchemaItem):
     ) -> None: ...
     def create_all(
         self,
-        bind: Optional[Any] = ...,
-        tables: Optional[Any] = ...,
+        bind: Optional[Union[Engine, Connection]] = ...,
+        tables: Optional[List[Table]] = ...,
         checkfirst: bool = ...,
     ) -> None: ...
     def drop_all(
         self,
-        bind: Optional[Any] = ...,
-        tables: Optional[Any] = ...,
+        bind: Optional[Union[Engine, Connection]] = ...,
+        tables: Optional[List[Table]] = ...,
         checkfirst: bool = ...,
     ) -> None: ...
 
 class ThreadLocalMetaData(MetaData):
     __visit_name__: str = ...
-    context: Any = ...
+    context: util.threading.local = ...
+    bind: Optional[Union[Engine, Connection]] = ...
     def __init__(self) -> None: ...
-    def bind(self): ...
-    bind: Any = ...
-    def is_bound(self): ...
+    def is_bound(self) -> bool: ...
     def dispose(self) -> None: ...
 
 class Computed(FetchedValue, SchemaItem):
     __visit_name__: str = ...
-    sqltext: Any = ...
-    persisted: Any = ...
-    column: Any = ...
+    sqltext: ClauseElement = ...
+    persisted: Optional[bool] = ...
+    column: Optional[Column[Any]] = ...
     def __init__(
-        self, sqltext: Any, persisted: Optional[Any] = ...
+        self,
+        sqltext: Union[str, ClauseElement],
```

Comment:
in other places sqltext was typed as `str` only.

Maybe all should be `str | TextClause` ?